README.md

```diff
@@ -266,6 +266,8 @@ The server can be configured with several environment variables.
 * `STATSD_PORT`: Port of statsd server (no default);
 * `STATSD_PREFIX`: Optional prefix to use with statsd metrics (no default).
 * `STATSD_USE_TCP`: If non-empty, use tcp instead of udp (no default).
+* `PROMETHEUS_HTTP_PORT`: Port to serve prometheus metrics from (no default).
```

Comment:
We might find other uses for the HTTP service beyond prometheus, so minor suggestion to refer to this as something more general (eg `HTTP_SERVICE_PORT`).